src/test/java/tests/RegisterTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import common.Constant;
+import helper.DataHelper;
+import helper.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_object.RegisterPage;
+
+public class RegisterTest extends BaseTest {
+    RegisterPage registerPage = new RegisterPage();
+    String email = "";
+    String pid = "";
+
+    @Test(description = "User can create new account")
+    public void tc07_UserCanCreateNewAccount() {
+        LogHelper.info("Click register tab");
+        registerPage.clickRegisterTab();
+
+        LogHelper.info("Enter valid information into all fields and click Register button");
+        email = DataHelper.getRandomEmail();
+        pid = DataHelper.getRandomPID(9);
+        registerPage.register(email, Constant.USERNAME, Constant.USERNAME, pid);
```

Comment:
Why did you use Constant.USERNAME instead of Constant.PASSWORD at Password field?

Reply:
fixed